manifests/params_agent.pp

```diff
@@ -174,8 +182,10 @@
       $ossec_syscheck_auto_ignore = undef
       $ossec_syscheck_directories_1 = '/etc,/usr/bin,/usr/sbin'
       $ossec_syscheck_directories_2 = '/bin,/sbin,/boot'
-      $ossec_syscheck_whodata = '"no"'
-      $ossec_syscheck_realtime = '"no"'
+      $ossec_syscheck_whodata_directories_1 = '"no"'
```

Comment:
Why double quoting? Please change it to `"no"` and fix recurrences

Reply:
Hi @jm404 !

Changes are applied and tested.

Kr,

Rshad